ledger/ledger-on-memory/BUILD.bazel

```diff
@@ -219,6 +210,9 @@ conformance_test(
     test_tool_args = [
         "--verbose",
         "--include=ParticipantPruningIT",
+        # The following tests don't actually care about multi-participant but they do need append-only.
+        "--additional=AppendOnlyCompletionDeduplicationInfoITCommandService",
+        "--additional=AppendOnlyCompletionDeduplicationInfoITCommandSubmissionService",
```

Comment:
The test after this one i.e. "conformance-test-multi-participant-command-deduplication" is an append-only test but asserts KVCommandDeduplicationIT

Reply:
@nicu-da Shouldn't it?

Reply:
Yes, we can add it here and remove `conformance-test-multi-participant-command-deduplication`

Reply:
Removed it for now (also, it was running just that) baa8be4

Reply:
@nicu-da Can we theoretically run `KVCommandDeduplicationIT` for both append-only and mutable schema? Does it make sense to?

Reply:
Added, it seems to work 35a274c